standalone-e2e-test/src/test/java/GetAllUsersTest.java

```diff
@@ -0,0 +1,75 @@
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetAllUsersTest extends AbstractRestAssuredBase {
```

Comment:
You can simply convert these tests to use spring using the @ExtendsWith annotation.
Sample: https://www.concretepage.com/spring-5/extendwith-springextension-spring-test
In the class used in the @ContextConfiguration annotation you can use @ComponentScan and also @PropertySource to load properties from files.